unfoldingWord/translationCore
# Bugfix bspidel 3072

#### This pull request addresses:

Added a period to dialog text.



#### How to test this pull request:
Visual inspection
1 Login as Guest
2 Select project
3 Click on ellipsis
4 Select "Upload to Door43

Note end of notice


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3140)
